src/langs/en.json

```diff
@@ -4,5 +4,18 @@
   "Settings": "Settings",
   "Search for models…": "Search for models…",
   "Preview": "Preview",
-  "Select Model": "Select Model"
+  "Select Model": "Select Model",
+  "Number of actions over time": "Number of actions over time",
+  "Intervals": "Intervals",
+  "week": "Week",
+  "day": "Day",
+  "Month": "Month",
```

Comment:
This one is capitalized while the others are not, it looks like it will not match any values you give. Open the console and watch out for un-translated keys.